pyomo/contrib/fbbt/fbbt.py

```diff
@@ -912,11 +899,17 @@ def fbbt_block(m, tol=1e-4, deactivate_satisfied_constraints=False, update_varia
                 var_ubs[v] = value(v.ub)
             var_to_con_map[v].append(c)
 
+    for _v in m.component_data_objects(ctype=Var, active=True, descend_into=True, sort=True):
+        if _v.is_fixed():
+            _v.setlb(_v.value)
+            _v.setub(_v.value)
```

Comment:
Why are you changing the actual variable bounds here?

Reply:
@jsiirola This is just an implementation detail. First, the fbbt functions are expected to change variable bounds. Second, suppose we have a constraint such as ```x + y == 1``` and ```x``` is fixed to ```1``` but does not have any bounds. In this case, we should be able to infer that ```y``` is ```0```. However, because of the way I have implemented fbbt, no bounds on y will be inferred unless the bounds on x get updated. It is certainly possible to do this correctly without changing the bounds on fixed variables, but I don't see a good reason not to change the bounds on fixed variables. If you can think of a reason, please let me know.